opensrp-chw/src/main/java/org/smartregister/chw/activity/PncMemberProfileActivity.java

```diff
@@ -376,17 +378,8 @@ private void addPncReferralTypes() {
                 BuildConfig.USE_UNIFIED_REFERRAL_APPROACH ? JSON_FORM.getPncUnifiedReferralForm() : JSON_FORM.getPncReferralForm(), CoreConstants.TASKS_FOCUS.PNC_DANGER_SIGNS));
 
         if (BuildConfig.USE_UNIFIED_REFERRAL_APPROACH) {
-            referralTypeModels.add(new ReferralTypeModel(getString(R.string.suspected_malaria),
-                    JSON_FORM.getMalariaReferralForm(), CoreConstants.TASKS_FOCUS.SUSPECTED_MALARIA));
-
-            referralTypeModels.add(new ReferralTypeModel(getString(R.string.hiv_referral),
-                    JSON_FORM.getHivReferralForm(), CoreConstants.TASKS_FOCUS.SUSPECTED_HIV));
-
-            referralTypeModels.add(new ReferralTypeModel(getString(R.string.tb_referral),
-                    JSON_FORM.getTbReferralForm(), CoreConstants.TASKS_FOCUS.SUSPECTED_TB));
-
             referralTypeModels.add(new ReferralTypeModel(getString(R.string.gbv_referral),
-                    JSON_FORM.getGbvReferralForm(), CoreConstants.TASKS_FOCUS.SUSPECTED_GBV));
+                    org.smartregister.chw.util.Constants.JSON_FORM.getGbvReferralForm(), CoreConstants.TASKS_FOCUS.SUSPECTED_GBV));
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'org.smartregister.chw.util.Constants.JSON_FORM.getGbvReferralForm' due to existing static import 'org.smartregister.chw.util.Constants.JSON_FORM'](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5590665)